avocado/utils/memory.py

```diff
@@ -320,3 +320,16 @@ def get_thp_value(feature):
         return (re.search(r"\[(\w+)\]", value)).group(1)
     else:
         return value
+
+
+def memtotal_sys():
+    """
+    get the total memory as detected by system
```

Comment:
@ldoktor Tnx, the real usage right now for me in avocado-vt, in process of sending PR there aswell, thought it would be useful here. 
`/proc/meminfo` shows [physical memory- kernel house keeping](not sure of real term though),
for my test this difference is too high when I boot Vm with TBs of memory, so wanted to have method to get a near real memory, and got to know this way can be used.